app/assets/stylesheets/task.scss

```diff
@@ -3,15 +3,33 @@
 
   background-color: $primary-bg-color;
   border: none;
+}
+
+.task-summary {
+  padding: 0.7rem 1.2rem;
+  border-bottom: solid 1px $collection-border-color;
 
-  a.collection-item {
-    @extend .z-depth-0;
+  color: $primary-font-color;
+  font-size: 1.3rem;
+
+  .tags {
+    margin-bottom: 0.8rem;
+  }
 
-    background-color: $primary-bg-color;
+  .chip {
+    background-color: #E6E18F;
```

Comment:
Color literals like `#E6E18F` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#E6E18F` should be written as `#e6e18f`